packages/cli/api-importers/v2-importer-commons/src/AbstractConverterContext.ts

```diff
@@ -348,6 +348,17 @@ export abstract class AbstractConverterContext<Spec extends object> {
         return value;
     }
 
+    public generateUniqueName(prefix: string, existingNames: string[]): string {
```

Comment:
nit: can we make sure all functions take in named arguments: `{ prefix, breadcrumbs}: { prefix: string; breadcrumbs: string[] }`